services/singleuser/Dockerfile

```diff
@@ -1,13 +1,34 @@
 # Build as renku/singleuser
 # Run with the DockerSpawner in JupyterHub
 
-ARG BASE_IMAGE=renku/minimal-notebook:latest
+ARG BASE_IMAGE=jupyter/base-notebook:latest
 FROM $BASE_IMAGE
 MAINTAINER Swiss Data Science Center <info@datascience.ch>
 
+
 ADD install /tmp/install
 ARG JUPYTERHUB_VERSION=0.8.1
 
+USER root
+# Copied from jupyter-minimal-notebook
+# Install all OS dependencies for fully functional notebook server
+RUN apt-get update && apt-get install -yq --no-install-recommends \
+    build-essential \
+    emacs \
+    git \
+    inkscape \
+    jed \
+    libsm6 \
+    libxext-dev \
+    libxrender1 \
+    lmodern \
+    netcat \
+    python-dev \
+    unzip \
+    nano \
+    && apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
```

Comment:
the user must be changed back to jovyan

Reply:
I had initially done that but there seemed to be an issue. I'll try again